src/gf.c

```diff
@@ -2511,8 +2513,11 @@ jl_code_instance_t *jl_compile_method_internal(jl_method_instance_t *mi, size_t
         }
     }
 
+    double compile_time = jl_hrtime();
     int did_compile = 0;
     codeinst = jl_generate_fptr(mi, world, &did_compile);
+    compile_time = jl_hrtime() - compile_time;
```

Comment:
I'm surprised the original PR had to add a measurement here... We  already have a metric for compilation time -- why isn't this incremented there?

Reply:
For example, here: https://github.com/JuliaLang/julia/blob/fe864374d50b121aeb3ce5473c206338b21df5e5/src/gf.c#L4390-L4396

Reply:
Asked here: https://github.com/JuliaLang/julia/pull/54662#pullrequestreview-2388882034